prov/efa/src/efa_av.c

```diff
@@ -728,6 +728,31 @@ static int efa_av_lookup(struct fid_av *av_fid, fi_addr_t fi_addr,
 	return 0;
 }
 
+/*
+ * @brief remove a set of addresses from AV and release its resources
+ *
+ * This function implements fi_av_remove() for EFA provider.
+ *
+ * Note that even after an address was removed from AV, it is still
+ * possible to get TX and RX completion for the address. Per libfabric
```

Comment:
This isn't true - the behavior is undefined. If we can handle av remove with in flight ops by marking tx pkts in progress with FI_ADDR_NOTAVAIL/with connid in all headers so be it, but I do think we have more flexibility here in the av remove path than this commit suggests.